bonnie-rest/src/main/java/com/cgi/bonnie/bonnierest/UserRegister.java

```diff
@@ -0,0 +1,25 @@
+package com.cgi.bonnie.bonnierest;
+
+import com.cgi.bonnie.businessrules.Role;
+import com.cgi.bonnie.businessrules.user.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "/user")
+public class UserRegister {
+
+    final private UserService userService;
+
+    @Autowired
+    public UserRegister(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping(path = "/register/{username}/{email}")
```

Comment:
This shouldn't be a GET. A GET request should not have any side effect. See: https://www.baeldung.com/logout-get-vs-post

Reply:
And I also don't like the idea of email address in the path

Reply:
Totally agree, just didn't find a way to accept post requests yet